src/Raven.Server/Web/System/AdminDatabasesHandler.cs

```diff
@@ -283,25 +284,27 @@ public Task Get()
             using (ServerStore.ContextPool.AllocateOperationContext(out TransactionOperationContext context))
             {
                 var json = await context.ReadForMemoryAsync(RequestBodyStream(), "read-conflict-resolver");
```

Comment:
Here we will find out about the garbage when we deserialize it when executing the command.